src/components/tabs/buy-tab-content/buy.overview.tsx

```diff
@@ -14,26 +14,51 @@ export function BuyTabContentOverview({ onAssetClicked }: BuyTabContentOverviewP
   const { assets } = useAssetContext();
   const { toHeader, toProtocol } = useBlockchain();
   const { addContract } = useMetaMask();
+  const availableChains: Blockchain[] = [
+    Blockchain.ETHEREUM,
+    Blockchain.BINANCE_SMART_CHAIN,
+    Blockchain.ARBITRUM,
+    Blockchain.OPTIMISM,
+    Blockchain.POLYGON,
+  ];
+
+  const blockchainToOrder: Record<Blockchain, number> = {
+    [Blockchain.ETHEREUM]: 0,
+    [Blockchain.ARBITRUM]: 1,
+    [Blockchain.BINANCE_SMART_CHAIN]: 2,
+    [Blockchain.OPTIMISM]: 3,
+    [Blockchain.POLYGON]: 4,
+    [Blockchain.DEFICHAIN]: Infinity,
+    [Blockchain.BITCOIN]: Infinity,
+    [Blockchain.CARDANO]: Infinity,
+  };
+
+  function orderByBlockchain(a: [Blockchain, Asset[]], b: [Blockchain, Asset[]]): number {
+    return blockchainToOrder[a[0]] - blockchainToOrder[b[0]];
+  }
```

Comment:
In the current exchange, the blockchain ordering can be done in DB (by setting the sort order on the assets). Is this fixed blockchain order required?

Reply:
AFAIK it was based on enum order of exchanges Blockchain enum and within a blockchain you could use sort order. But could be wrong. At least the order was different between DEV and PRD and therefore I have changed it to look like PRD.